mesheryctl/internal/cli/root/perf/apply.go

```diff
@@ -261,8 +278,39 @@ var applyCmd = &cobra.Command{
 	},
 }
 
+func multipleProfileConfirmation(profiles []models.PerformanceProfile) int {
+	reader := bufio.NewReader(os.Stdin)
+
+	for index, a := range profiles {
+		fmt.Printf("Index: %v\n", index)
+		fmt.Printf("Name: %v\n", a.Name)
+		fmt.Printf("ID: %s\n", a.ID.String())
+		fmt.Printf("Endpoint: %v\n", a.Endpoints[0])
+		fmt.Printf("Load Generators: %v\n", a.LoadGenerators[0])
+		fmt.Println("---------------------")
+	}
+
+	for {
+		fmt.Printf("Enter the index of profile?? ")
```

Comment:
I think we have a helper function to read inputs from the user. Not sure though. It might be something we should add.

Replace "??" with "?"

Reply:
Definitely, we have the function for reading inputs from there, but this needed a separate function here. I'll replace that.

That token thing isn't an issue here, I'll implement changes regarding that in the system-check PR.